kernel/src/main/scala/cats/kernel/Order.scala

```diff
@@ -139,7 +139,16 @@ abstract class OrderFunctions[O[T] <: Order[T]] extends PartialOrderFunctions[O]
     ev.max(x, y)
 }
 
-object Order extends OrderFunctions[Order] {
+trait OrderToOrderingConversion {
+  /**
+   * Implicitly convert a `Order[A]` to a `scala.math.Ordering[A]`
```

Comment:
I agree, but I changed this comment. I am not seeing how it showed up here :stuck_out_tongue: